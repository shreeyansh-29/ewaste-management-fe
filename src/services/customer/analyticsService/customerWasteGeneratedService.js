import {CUSTOMER_ANALYTICS_V1} from "../../../container/constant/constant";
import api from "../../../core/utilities/httpProvider";

export const customerWasteGeneratedService = () => {
  return api.get(CUSTOMER_ANALYTICS_V1);
};
