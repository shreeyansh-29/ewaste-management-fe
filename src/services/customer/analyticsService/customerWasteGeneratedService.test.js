/* eslint-disable no-undef */

import {customerWasteGeneratedService} from "./customerWasteGeneratedService";
import React from "react";
import Enzyme, {shallow} from "enzyme";
import toJson from "enzyme-to-json";
import Adapter from "@wojtekmaj/enzyme-adapter-react-17";

Enzyme.configure({adapter: new Adapter()});

it("testing customerWasteGeneratedService", () => {
  const Service = customerWasteGeneratedService;
  const wrapper = shallow(<Service />);
  expect.assertions(1);
  expect(toJson(wrapper)).toMatchSnapshot();
});
