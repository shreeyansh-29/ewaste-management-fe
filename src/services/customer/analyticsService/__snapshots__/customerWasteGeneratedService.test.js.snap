// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing customerWasteGeneratedService 1`] = `<Component />`;
